hardhat.config.ts

```diff
@@ -40,14 +49,22 @@ const config: HardhatUserConfig = {
     wallets: {
       OpenZeppelin: {
         accountName: "OpenZeppelin",
-        modulePath: "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount",
-        accountPath: "~/.starknet_accounts"
-      }
-    }
+        modulePath:
+          "starkware.starknet.wallets.open_zeppelin.OpenZeppelinAccount",
+        accountPath: "~/.starknet_accounts",
+      },
+    },
   },
   networks: {
     devnet: {
-      url: "http://localhost:5000"
+      url: "http://localhost:5000",
+    },
+    fork: {
```

Comment:
we don't understand which layer corresponds to this network. you can name it `l1-testnet` instead